apps/web-giddh/src/app/vouchers/list/list.component.ts

```diff
@@ -1149,31 +1173,7 @@ export class VoucherListComponent implements OnInit, OnDestroy {
      * @memberof VoucherCreateComponent
      */
     private getInvoiceSettings(): void {
-        this.componentStore.invoiceSettings$.pipe(takeUntil(this.destroyed$)).subscribe(settings => {
```

Comment:
why we changed this logic? It will always call invoice settings api even if we have the data in store.

Reply:
If we navigate to the invoice page, I need to refresh the invoiceSetting API response.